bodhi/views/generic.py

```diff
@@ -193,6 +194,25 @@ def new_override(request):
     return dict(nvr=nvr)
 
 
+@view_config(route_name='popup_toggle', request_method='POST')
+def popup_toggle(request):
+    # Get the user
+    from bodhi.models import User
+    userid = authenticated_userid(request)
+    if userid is None:
+        raise HTTPForbidden("You must be logged in.")
+    user = request.db.query(User).filter_by(name=unicode(userid)).first()
```

Comment:
Nah, ``request.user`` is a ``munch`` object now.  So, we can *get* the ``show_popups`` value from it, but we can't *set* it.